src/include/c_api/kuzu.h

```diff
@@ -292,10 +292,11 @@ typedef enum {
     // variable size types
     KUZU_STRING = 50,
     KUZU_BLOB = 51,
-    KUZU_VAR_LIST = 52,
-    KUZU_STRUCT = 53,
-    KUZU_MAP = 54,
-    KUZU_UNION = 55,
+    KUZU_UUID = 52,
```

Comment:
I think you have to add some test cases for the c api, and getter/setter functions. You can leave the API related stuff in the next PR